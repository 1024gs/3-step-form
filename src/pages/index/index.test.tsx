import React from '../../../vendors/react';
import {shallow} from 'enzyme';
import IndexPage from './index';

describe('IndexPage', () => {
  it('renders', () => {
    const wrapper = shallow(<IndexPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
