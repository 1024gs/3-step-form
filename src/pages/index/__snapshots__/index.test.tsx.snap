// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders 1`] = `
<div
  className="app"
>
  <Navbar />
  <br />
  <div
    className="container"
  >
    <FormNav />
    <br />
    <form
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label
          htmlFor="name"
        >
          Name: *
        </label>
        <input
          className="form-control"
          id="name"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="role"
        >
          Role:
        </label>
        <input
          className="form-control"
          id="role"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="email"
        >
          Email: *
        </label>
        <input
          className="form-control"
          id="email"
          onChange={[Function]}
          pattern=".+@.+"
          required={true}
          type="email"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="password"
        >
          Password: *
        </label>
        <input
          className="form-control"
          id="password"
          minLength="10"
          onChange={[Function]}
          pattern="^(?=.*[a-z])(?=.*[A-Z])(?=.*\\\\d)[a-zA-Z\\\\d]+$"
          required={true}
          type="password"
          value=""
        />
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
